styles/body.css

```diff
@@ -0,0 +1,126 @@
+html {
+    font-family: sans-serif;
+    font-size: 14px;
+    color: #2c2b25;
+}
+
+.banner {
+    padding: 2.5vh 2.5vw;
+    margin: 2vh 2.5vw;
+    background-size: cover;
+    background-position: center;
+}
+
+.picture {
```

Comment:
try to use BEM methodologies for CSS naming. this means as example for:
<div class="div1">
<div class="div1-div2">
</div></div>

.div1 {}
.div1-div2 {} 

You have to rename the picture class to banner-picture since it is referring to picture from the banner